.github/ISSUE_TEMPLATE/unstable-test.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Unstable Test
+about: Associate issue for an unstable test that was skipped.
+title: "Unstable: "
+labels: testing, 0 - new, p - high
+assignees: ""
+---
+
+## Summary
+
+**Which component?:**
+
+**Which test(s)?:** <!-- list unstable test(s) or suite -->
+
+**PR that skipped the test:** #
```

Comment:
maybe we should have a section for the build URL where the test failed?

Reply:
That's a great idea.

Reply:
Updated issue template and also added a section in the PR template.